Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG MB_EDITION=oss
 WORKDIR /home/circleci
 
 COPY --chown=circleci . .
-RUN NODE_ENV=production MB_EDITION=$MB_EDITION yarn --frozen-lockfile && \
+RUN WEBPACK_BUNDLE=production MB_EDITION=$MB_EDITION yarn --frozen-lockfile && \
```

Comment:
@paoliniluis I think this can now be simplified as this one, wdyt?
```
RUN CI=true MB_EDITION=$MB_EDITION bin/build frontend
```

Reply:
Agree!

Reply:
way easier, of course